app/components/buySell/BuySellDialog.js

```diff
@@ -152,7 +158,6 @@ export default class BuySellDialog extends Component<Props, State> {
           <div className={styles.component}>
             <div className={styles.description}>
               {intl.formatMessage(messages.dialogDescription)}
-              {this.props.walletList}
```

Comment:
@nicarq I delete this since it didn't make sense (`this.props.walletList` here is an async function) and it didn't seem necessary for the UI to work properly

Reply:
yeah, I missed it. We definitely don't need this here

Reply:
thanks Seba!